packages/node_modules/@webex/react-component-utils/src/adaptiveCardUtil.js

```diff
@@ -100,11 +103,17 @@ export function getAdaptiveCard(
     const adaptiveCard = new PatchedAdaptiveCard.AdaptiveCard();
 
     adaptiveCard.onExecuteAction = (action) => {
+      const btnClicked = action.renderedElement;
+
       switch (action.getJsonTypeName()) {
         case ADAPTIVE_CARD_OPEN_URL_ACTION:
           window.open(action.url);
           break;
         case ADAPTIVE_CARD_SUBMIT_ACTION: {
+          btnClicked.disabled = true;
+          setTimeout(() => {
+            btnClicked.disabled = false;
+          }, 2500);
```

Comment:
Is there no way of having the submission task re-enable the button after it has finished instead of guess how long it will take to submit the action?

Reply:
`onExecuteAction` function executes only when there is any action. I didn't find any other better approach. If you have any please suggest me.